### PR TITLE
insertMany nosqlbench workload

### PR DESCRIPTION
**Pull request by Yuqi-Du**
**What this PR does**:
added an insertMany nosqlbench workload 

**Checklist**
- [x] Changes manually tested
- [x] Automated Tests added/updated
- [x] Documentation added/updated
- [x] CLA Signed: [DataStax CLA](https://cla.datastax.com/)

### CODE REVIEW

File: nosqlbench/http-jsonapi-vector-insertmany.yaml [hunk 1]
```diff
@@ -92,103 +82,248 @@ blocks:
...
-          ratio: 5
-        space: "{space}"
+      insertMany:
         method: POST
-        uri: <<protocol:http>>://{weighted_hosts}:<<jsonapi_port:8181>><<path_prefix:>>/v1/<<namespace:jsonapi_vector_crud_namespace>>/<<collection:jsonapi_vector_crud_collection>>
+        uri: <<protocol:http>>://{weighted_hosts}:<<jsonapi_port:8181>><<path_prefix:>>/v1/<<namespace:TEMPLATE(keyspace,baselines)>>/<<collection:TEMPLATE(table,keyvalue)>>
         Accept: "application/json"
         X-Cassandra-Request-Id: "{request_id}"
-        Token: "{token}"
+        X-Cassandra-Token: "{token}"
```
Comment (jeffreyscarpenter) [inline nosqlbench/http-jsonapi-vector-insertmany.yaml:93]: didn't we change to token?

Comment (jeffreyscarpenter) [reply nosqlbench/http-jsonapi-vector-insertmany.yaml:93]: and other places

**Review by jeffreyscarpenter — APPROVED** [review]
Looks good overall, one comment on token name

File: nosqlbench/http-jsonapi-vector-insertmany.yaml (hunk 1 above)
Comment (Yuqi-Du) [reply nosqlbench/http-jsonapi-vector-insertmany.yaml]: you are sharp. solved

**Review by jeffreyscarpenter — APPROVED** [review]

**Pull request merged**